kwamp-client-core/src/main/kotlin/com/laurencegarmstrong/kwamp/client/core/Client.kt

```diff
@@ -153,6 +157,11 @@ class ClientImpl(
         topicPattern: UriPattern,
         eventHandler: EventHandler
     ) = subscriber.subscribe(topicPattern, eventHandler)
+
```

Comment:
To be consistent with how other functions have been laid out, I think we could just have one newline here.
```suggestion
```